test_deepdoc_env.py

```
@@ -0,0 +1,105 @@
+"""
+DeepDoc 虚拟环境测试脚本
+
+使用方法:
+    1. 激活虚拟环境: .venv\Scripts\activate (Windows) 或 source .venv/bin/activate (Linux/Mac)
```

Ruff (0.16.1)

[warning] Invalid escape sequence: `\S`

Add backslash to escape sequence

(W605)